package.json

```diff
@@ -9,7 +9,7 @@
     "build:dev": "webpack --mode=development",
     "build:prod": "webpack --mode=production --node-env=production",
     "watch": "webpack --watch",
-    "serve": "webpack serve",
+    "serve": "webpack serve --mode=development",
```

Comment:
I forgot I added this, but this is what will set the MODE variable in our .env file to 'development' or 'production' so we can change our API call URLs based on which mode we are on.